data/src/client.rs

```diff
@@ -1519,7 +1519,7 @@ fn remove_tag(key: &str, tags: &mut Vec<irc::proto::Tag>) -> Option<String> {
 fn start_reroute(command: &Command) -> bool {
     use Command::*;
 
-    matches!(command, WHO(..) | WHOIS(..) | WHOWAS(..))
+    matches!(command, WHO(..) | WHOIS(..) | WHOWAS(..) | MODE(..))
```

Comment:
For this to only apply to user `MODE` commands, we should look at the `target` like 
```rust
    if let MODE(target, _, _) = command {
        !proto::is_channel(target)
    } else {
        matches!(command, WHO(..) | WHOIS(..) | WHOWAS(..))
    }
```

Reply:
Understood, done.